Makefile

```diff
@@ -0,0 +1,12 @@
+SHELL := /bin/bash
+
+help:
+	@echo "Usage:"
+	@echo "    make release    | Release to pypi."
+
+release:
+	python setup.py register sdist bdist_wheel upload
+
+test:
+	coverage run -m unittest
```

Comment:
It's equivalent to [`python -m`](https://docs.python.org/3.5/using/cmdline.html#cmdoption-m), except it runs coverage.

`--module` works too though.